RealmSwift/Map.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  File.swift
+//  RealmSwift
+//
+//  Created by Pavel Yakimenko on 27/01/2021.
+//  Copyright © 2021 Realm. All rights reserved.
+//
+
+import Foundation
+import Realm
+import Realm.Private
+
+/**
+ * Key-value collection. Where the key is a string and value is one of the available Realm types.
+ * We use Map to don't intefere with the native Swift's Dictionary type.
+ */
```

Comment:
/**
 `Map` is a key-value storage container used to store supported Realm types.

 `Map` is a generic type that is parameterized on the type it stores. This can be either an `Object`
 subclass or one of the following types: `Bool`, `Int`, `Int8`, `Int16`, `Int32`, `Int64`, `Float`, `Double`,
 `String`, `Data`, `Date`, `Decimal128`, and `ObjectId` (and their optional versions)

`Map` only supports `String` as a key.

 Unlike Swift's native collections, `Map` is a reference types, and are only immutable if the Realm that manages them
 is opened as read-only.

 A `Map` can be filtered and sorted with the same predicates as `Results<Element>`.

 Properties of `Map` type defined on `Object` subclasses must be declared as `let` and cannot be `dynamic`.
 */

Reply:
nice, thanks